handbook.md

```diff
@@ -742,22 +743,23 @@ The goal of Atlantian Youth Rapier is to recreate pre-17th century fencing.  All
         #. Fighters with a valid authorization from another kingdom who move into Atlantia may trade in for an Atlantian authorization card. See the Policies of the MoL for the procedure.
     #. The Kingdom Earl Marshal or Deputy Earl Marshal for Rapier Combat shall warrant a select group of marshals for newly approved authorization forms.  Future authorizations shall follow the standard authorization policies and procedures.
 #. Rapier Combat Weapons and Armor Standards
-    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html),](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
-        #. No spears are allowed.
+    #. Atlantia follows the Society for Creative Anachronism, Incorporated's "Fencing Marshals’ Handbook" (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html)](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
+        #. Only rubber-headed spears are allowed.
         #. Fighters who choose to participate in Cut & Thrust combat acknowledge that they may be struck with a harder blow than they would normally receive in rapier or light rapier combat, and they should act with the interest of the safety of their opponent in mind.
-        #. In addition to the Society requirements, for Cut & Thrust combat, the back of the head must also be covered by rigid material, lined by at least 1/4 inch (6mm) of closed cell foam or an equivalent padding.
         #. In addition to the Society requirements, all Rapier and Cut & Thrust blade tips must include adequate protection against punch-through of the blade. This may include:
             * A metal disc (such as a washer) placed between the end of the blade and the tip.
-            * A nut or other blunt metal object welded to the end of the blade in compliance with the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated.
+            * A nut or other blunt metal object welded to the end of the blade in compliance with the Society for Creative Anachronism, Incorporated's "Fencing Marshals’ Handbook".
+            * A properly sized Darkwood Rapier tip installed following manufacturer's installation instructions meets this requirement.
             * Any approach specifically approved by the Deputy Earl Marshal for Rapier Combat.
```

Comment:
I would like to see an additional provision to allow flared (or rolled) tips as an acceptable protection against punch-through like what you see commercially available by companies like castille.

Reply:
This is something to keep in mind for later.  I have concerns having seen some of the flared tip failures in WMA circles, this will take extra investigation. 

Closing for now, as these rules have been submitted to TRM.

Reply:
Are those flared tips alone or flared tips punching through rubber tips? What kind of failures? I'm not advocating losing the rubber tips, just that a sufficiently flared tip creates an increased surface area like a nut or washer (whose minimum diameter is not specified).

Reply:
I am happy to investigate this, but not as part of this rules update.